pkg/defaults/policies/files/deployment_has_ingress_network_policy.json

```diff
@@ -141,6 +141,42 @@
       ]
     }
   ],
+  "mitreAttackVectors": [
+    {
+      "tactic": "TA0001",
+      "techniques": [
+        "T1133",
+        "T1199"
+      ]
+    },
+    {
+      "tactic": "TA0005",
+      "techniques": [
+        "T1610"
```

Comment:
Nitpick: Agree on T1610, but should we maybe sort this in Execution (T0002) instead of Defense Evasion (T0005)?